### ISSUE TITLE
[rtl/rstmgr] Make cpu and alert info layout known to sw

### ISSUE THREAD

**Issue by matutem** [lowRISC/opentitan#8945]
The alert_info contents is defined in alert_pkg::alert_crashdump_t, and cpu_info in ibex_pkg::crash_dump_t. The actual layout and size of these need to be known to software if we are ever to make use of these dumps. Maybe it can be piped via hjson?

**Comment by rswarbrick** [comment]
Yep, I was imagining enhancing topgen. I don't think we've got a notion of "import" in reggen and I'm not sure that the crash dump structure would necessarily be specified in "reggen format" anyway.

**Comment by matutem** [comment]
The declaration of cpu_dump belongs to the ibex, and that of alert_dump to alert_handler, so this is not really a rstmgr request: all rstmgr needs to know is the total number of bits in each.
